With this change, the following naming conventions are still being reported in the existing code:

```
pkg/volume/azure_file/azure_file.go:20:1: ST1003: should not use underscores in package names (stylecheck)
pkg/volume/azure_file/azure_provision.go:20:1: ST1003: should not use underscores in package names (stylecheck)
pkg/volume/azure_file/azure_util.go:20:1: ST1003: should not use underscores in package names (stylecheck)
pkg/volume/azure_file/doc.go:19:1: ST1003: should not use underscores in package names (stylecheck)
pkg/volume/azure_file/azure_file_test.go:20:1: ST1003: should not use underscores in package names (stylecheck)
test/e2e/storage/drivers/csi-test/mock/service/controller.go:739:6: ST1003: func getSnapshotById should be getSnapshotByID (stylecheck)
test/e2e/storage/drivers/csi-test/mock/service/controller.go:763:6: ST1003: func getSnapshotByVolumeId should be getSnapshotByVolumeID (stylecheck)
test/e2e/storage/drivers/csi-test/mock/service/service.go:255:37: ST1003: method parameter sourceVolumeId should be sourceVolumeID (stylecheck)
test/e2e_node/testing-manifests/embed.go:17:1: ST1003: should not use underscores in package names (stylecheck)
pkg/scheduler/internal/queue/scheduling_queue_test.go:1964:112: ST1003: func parameter wantInitialAttemptTs should be wantInitialAttemptTS (stylecheck)
pkg/kubelet/volumemanager/volume_manager.go:556:6: ST1003: func getExtraSupplementalGid should be getExtraSupplementalGID (stylecheck)
pkg/kubelet/volumemanager/volume_manager.go:567:10: ST1003: range var existingGid should be existingGID (stylecheck)
pkg/kubelet/volumemanager/volume_manager_test.go:272:2: ST1003: var existingGid should be existingGID (stylecheck)
pkg/apis/core/validation/validation.go:4652:4: ST1003: var mungeCpuMemResources should be mungeCPUMemResources (stylecheck)
pkg/volume/git_repo/doc.go:18:1: ST1003: should not use underscores in package names (stylecheck)
pkg/volume/git_repo/git_repo.go:17:1: ST1003: should not use underscores in package names (stylecheck)
pkg/volume/git_repo/git_repo_test.go:17:1: ST1003: should not use underscores in package names (stylecheck)
pkg/controller/volume/persistentvolume/metrics/metrics.go:275:2: ST1003: struct field startTs should be startTS (stylecheck)
test/instrumentation/main.go:46:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
test/instrumentation/main.go:47:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
pkg/registry/core/service/ipallocator/bitmap_test.go:577:2: ST1003: should not use underscores in Go names; var dynamic_allocated should be dynamicAllocated (stylecheck)
pkg/registry/core/service/ipallocator/bitmap_test.go:595:2: ST1003: should not use underscores in Go names; var static_errors should be staticErrors (stylecheck)
pkg/registry/core/service/ipallocator/bitmap_test.go:682:2: ST1003: should not use underscores in Go names; var static_allocated should be staticAllocated (stylecheck)
pkg/registry/core/service/ipallocator/bitmap_test.go:686:2: ST1003: should not use underscores in Go names; var static_errors should be staticErrors (stylecheck)
pkg/registry/core/service/ipallocator/bitmap_test.go:690:2: ST1003: should not use underscores in Go names; var dynamic_allocated should be dynamicAllocated (stylecheck)
pkg/registry/core/service/ipallocator/bitmap_test.go:694:2: ST1003: should not use underscores in Go names; var dynamic_errors should be dynamicErrors (stylecheck)
pkg/registry/core/service/ipallocator/ipallocator_test.go:502:2: ST1003: should not use underscores in Go names; var dynamic_allocated should be dynamicAllocated (stylecheck)
pkg/registry/core/service/ipallocator/ipallocator_test.go:520:2: ST1003: should not use underscores in Go names; var static_errors should be staticErrors (stylecheck)
staging/src/k8s.io/api/batch/v1/types.go:47:2: ST1003: const ControllerUidLabel should be ControllerUIDLabel (stylecheck)
staging/src/k8s.io/api/core/v1/resource.go:29:25: ST1003: method Cpu should be CPU (stylecheck)
staging/src/k8s.io/api/core/v1/types.go:6450:2: ST1003: const SecretTypeDockerConfigJson should be SecretTypeDockerConfigJSON (stylecheck)
staging/src/k8s.io/api/core/v1/types.go:6453:2: ST1003: const DockerConfigJsonKey should be DockerConfigJSONKey (stylecheck)
staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_discovery_controller.go:90:2: ST1003: var aggregatedApiResourcesForDiscovery should be aggregatedAPIResourcesForDiscovery (stylecheck)
staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go:510:22: ST1003: should not use underscores in Go names; method removeStorage_locked should be removeStorageLocked (stylecheck)
staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go:1295:14: ST1003: var foundApiVersion should be foundAPIVersion (stylecheck)
staging/src/k8s.io/apimachinery/pkg/test/apis_meta_v1_unstructed_unstructure_test.go:38:2: ST1003: var rawJson should be rawJSON (stylecheck)
staging/src/k8s.io/apimachinery/pkg/runtime/conversion.go:118:6: ST1003: should not use underscores in Go names; func string_to_int64 should be stringToInt64 (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/jsonmergepatch/patch.go:93:6: ST1003: func keepOrDeleteNullInJsonPatch should be keepOrDeleteNullInJSONPatch (stylecheck)
staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/labels_test.go:66:3: ST1003: should not use underscores in Go names; var ls_in should be lsIn (stylecheck)
staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/labels_test.go:67:3: ST1003: should not use underscores in Go names; var ls_out should be lsOut (stylecheck)
staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/labels_test.go:115:3: ST1003: should not use underscores in Go names; var ls_in should be lsIn (stylecheck)
staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/labels_test.go:116:3: ST1003: should not use underscores in Go names; var ls_out should be lsOut (stylecheck)
staging/src/k8s.io/apimachinery/pkg/runtime/codec_test.go:45:3: ST1003: struct field expectedId should be expectedID (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper.go:162:28: ST1003: method dialWithHttpProxy should be dialWithHTTPProxy (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go:33:29: ST1003: func parameter backendUrl should be backendURL (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go:74:30: ST1003: func parameter backendUrl should be backendURL (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go:126:2: ST1003: var backendUrlChan should be backendURLChan (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go:129:6: ST1003: var backendUrl should be backendURL (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go:137:2: ST1003: var proxyUrlChan should be proxyURLChan (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go:140:6: ST1003: var proxyUrl should be proxyURL (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch_test.go:6339:6: ST1003: func sortJsonOrFail should be sortJSONOrFail (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/net/http.go:323:2: ST1003: var hdrRealIp should be hdrRealIP (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/net/interface_test.go:66:7: ST1003: should not use underscores in Go names; const route_Invalidhex should be routeInvalidhex (stylecheck)
staging/src/k8s.io/apimachinery/pkg/util/net/interface_test.go:95:7: ST1003: should not use underscores in Go names; const v6route_Invalidhex should be v6routeInvalidhex (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/testing/conversion.go:26:6: ST1003: func convertUrlValuesToSimpleGetOptions should be convertURLValuesToSimpleGetOptions (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler_test.go:169:2: ST1003: should not use underscores in Go names; var res1_initial should be res1Initial (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler_test.go:170:2: ST1003: should not use underscores in Go names; var res2_initial should be res2Initial (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler_test.go:185:2: ST1003: should not use underscores in Go names; var res1_addedToOne should be res1AddedToOne (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler_test.go:186:2: ST1003: should not use underscores in Go names; var res2_addedToOne should be res2AddedToOne (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler_test.go:201:2: ST1003: should not use underscores in Go names; var res1_addedToBoth should be res1AddedToBoth (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler_test.go:202:2: ST1003: should not use underscores in Go names; var res2_addedToBoth should be res2AddedToBoth (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler_test.go:223:2: ST1003: should not use underscores in Go names; var res1_removeFromBoth should be res1RemoveFromBoth (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler_test.go:224:2: ST1003: should not use underscores in Go names; var res2_removeFromBoth should be res2RemoveFromBoth (stylecheck)
staging/src/k8s.io/apiserver/pkg/server/healthz/healthz_test.go:267:6: ST1003: func createGetRequestWithUrl should be createGetRequestWithURL (stylecheck)
staging/src/k8s.io/apiserver/pkg/server/config.go:784:2: ST1003: var genericApiServerHookName should be genericAPIServerHookName (stylecheck)
staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:557:2: ST1003: var stopHttpServerCh should be stopHTTPServerCh (stylecheck)
staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:561:3: ST1003: var timeToStopHttpServerCh should be timeToStopHTTPServerCh (stylecheck)
staging/src/k8s.io/apiserver/pkg/registry/generic/rest/response_checker.go:32:6: ST1003: type HttpResponseChecker should be HTTPResponseChecker (stylecheck)
staging/src/k8s.io/apiserver/pkg/registry/generic/rest/response_checker.go:42:6: ST1003: type GenericHttpResponseChecker should be GenericHTTPResponseChecker (stylecheck)
staging/src/k8s.io/apiserver/pkg/registry/generic/rest/response_checker.go:69:6: ST1003: func NewGenericHttpResponseChecker should be NewGenericHTTPResponseChecker (stylecheck)
staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go:217:6: ST1003: func UpdateEtcdDbSize should be UpdateEtcdDBSize (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/installer.go:53:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/apiserver/pkg/endpoints/installer.go:54:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/apiserver/pkg/authentication/group/token_group_adder_test.go:53:6: ST1003: func toJson should be toJSON (stylecheck)
staging/src/k8s.io/apiserver/plugin/pkg/audit/log/backend.go:34:2: ST1003: const FormatJson should be FormatJSON (stylecheck)
staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go:425:2: ST1003: var valueJson should be valueJSON (stylecheck)
staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/admission_test.go:252:2: ST1003: struct field keyId should be keyID (stylecheck)
staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/admission_test.go:1657:2: ST1003: var valueJson should be valueJSON (stylecheck)
staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/admission_test.go:1788:2: ST1003: var valueJson should be valueJSON (stylecheck)
staging/src/k8s.io/client-go/util/testing/fake_openapi_handler.go:35:2: ST1003: struct field HttpServer should be HTTPServer (stylecheck)
staging/src/k8s.io/client-go/rest/url_utils.go:82:6: ST1003: func defaultServerUrlFor should be defaultServerURLFor (stylecheck)
staging/src/k8s.io/client-go/rest/urlbackoff.go:36:16: ST1003: interface method parameter actualUrl should be actualURL (stylecheck)
staging/src/k8s.io/client-go/rest/urlbackoff.go:37:19: ST1003: interface method parameter actualUrl should be actualURL (stylecheck)
staging/src/k8s.io/client-go/rest/urlbackoff.go:52:35: ST1003: method parameter actualUrl should be actualURL (stylecheck)
staging/src/k8s.io/client-go/rest/urlbackoff.go:56:38: ST1003: method parameter actualUrl should be actualURL (stylecheck)
staging/src/k8s.io/client-go/rest/urlbackoff.go:73:22: ST1003: method baseUrlKey should be baseURLKey (stylecheck)
staging/src/k8s.io/client-go/rest/urlbackoff.go:86:36: ST1003: method parameter actualUrl should be actualURL (stylecheck)
staging/src/k8s.io/client-go/rest/urlbackoff.go:101:39: ST1003: method parameter actualUrl should be actualURL (stylecheck)
staging/src/k8s.io/client-go/rest/client_test.go:340:2: ST1003: var theUrl should be theURL (stylecheck)
staging/src/k8s.io/client-go/rest/request_test.go:1547:44: ST1003: method parameter actualUrl should be actualURL (stylecheck)
staging/src/k8s.io/client-go/rest/request_test.go:1550:47: ST1003: method parameter actualUrl should be actualURL (stylecheck)
staging/src/k8s.io/client-go/rest/request_test.go:3025:69: ST1003: method parameter actualUrl should be actualURL (stylecheck)
staging/src/k8s.io/client-go/rest/request_test.go:3033:66: ST1003: method parameter actualUrl should be actualURL (stylecheck)
staging/src/k8s.io/client-go/rest/urlbackoff_test.go:28:2: ST1003: var theUrl should be theURL (stylecheck)
staging/src/k8s.io/client-go/util/cert/cert_test.go:27:7: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/client-go/tools/cache/delta_fifo.go:306:21: ST1003: should not use underscores in Go names; method hasSynced_locked should be hasSyncedLocked (stylecheck)
staging/src/k8s.io/client-go/tools/cache/fifo.go:158:16: ST1003: should not use underscores in Go names; method hasSynced_locked should be hasSyncedLocked (stylecheck)
staging/src/k8s.io/client-go/tools/clientcmd/client_config.go:616:27: ST1003: func parameter masterUrl should be masterURL (stylecheck)
staging/src/k8s.io/client-go/tools/clientcmd/client_config.go:632:38: ST1003: func parameter masterUrl should be masterURL (stylecheck)
staging/src/k8s.io/cli-runtime/pkg/resource/builder.go:48:7: ST1003: const defaultHttpGetAttempts should be defaultHTTPGetAttempts (stylecheck)
staging/src/k8s.io/cli-runtime/pkg/resource/visitor.go:245:2: ST1003: struct field HttpAttemptCount should be HTTPAttemptCount (stylecheck)
staging/src/k8s.io/cli-runtime/pkg/resource/visitor.go:259:6: ST1003: func readHttpWithRetries should be readHTTPWithRetries (stylecheck)
staging/src/k8s.io/cli-runtime/pkg/resource/helper_test.go:88:3: ST1003: struct field HttpErr should be HTTPErr (stylecheck)
staging/src/k8s.io/cli-runtime/pkg/resource/helper_test.go:175:3: ST1003: struct field HttpErr should be HTTPErr (stylecheck)
staging/src/k8s.io/cli-runtime/pkg/resource/helper_test.go:276:3: ST1003: struct field HttpErr should be HTTPErr (stylecheck)
staging/src/k8s.io/cli-runtime/pkg/resource/helper_test.go:383:3: ST1003: struct field HttpErr should be HTTPErr (stylecheck)
staging/src/k8s.io/cli-runtime/pkg/resource/helper_test.go:569:3: ST1003: struct field HttpErr should be HTTPErr (stylecheck)
staging/src/k8s.io/code-generator/cmd/lister-gen/generators/lister.go:272:5: ST1003: should not use underscores in Go names; var typeListerInterface_NonNamespaced should be typeListerInterfaceNonNamespaced (stylecheck)
staging/src/k8s.io/code-generator/cmd/lister-gen/generators/lister.go:300:5: ST1003: should not use underscores in Go names; var typeLister_List should be typeListerList (stylecheck)
staging/src/k8s.io/code-generator/cmd/lister-gen/generators/lister.go:310:5: ST1003: should not use underscores in Go names; var typeLister_NamespaceLister should be typeListerNamespaceLister (stylecheck)
staging/src/k8s.io/code-generator/cmd/lister-gen/generators/lister.go:317:5: ST1003: should not use underscores in Go names; var typeLister_NonNamespacedGet should be typeListerNonNamespacedGet (stylecheck)
staging/src/k8s.io/code-generator/cmd/lister-gen/generators/lister.go:354:5: ST1003: should not use underscores in Go names; var namespaceLister_List should be namespaceListerList (stylecheck)
staging/src/k8s.io/code-generator/cmd/lister-gen/generators/lister.go:364:5: ST1003: should not use underscores in Go names; var namespaceLister_Get should be namespaceListerGet (stylecheck)
staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion.go:1090:25: ST1003: method generateFromUrlValues should be generateFromURLValues (stylecheck)
staging/src/k8s.io/component-base/cli/flag/ciphersuites_flag_test.go:29:3: ST1003: should not use underscores in Go names; struct field expected_error should be expectedError (stylecheck)
staging/src/k8s.io/csi-translation-lib/plugins/gce_pd_test.go:306:3: ST1003: struct field wantVolId should be wantVolID (stylecheck)
staging/src/k8s.io/kubectl/pkg/cmd/diff/diff_test.go:86:4: ST1003: should not use underscores in Go names; var output_msg should be outputMsg (stylecheck)
staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go:456:6: ST1003: func AddJsonFilenameFlag should be AddJSONFilenameFlag (stylecheck)
staging/src/k8s.io/kubectl/pkg/describe/describe.go:91:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/kubectl/pkg/describe/describe.go:92:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/kubectl/pkg/describe/describe.go:93:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/kubectl/pkg/describe/describe.go:94:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/kubectl/pkg/describe/describe.go:95:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/kubectl/pkg/describe/describe.go:4142:3: ST1003: var fractionCpuReq should be fractionCPUReq (stylecheck)
staging/src/k8s.io/kubectl/pkg/describe/describe.go:4143:3: ST1003: var fractionCpuLimit should be fractionCPULimit (stylecheck)
staging/src/k8s.io/kubectl/pkg/describe/describe.go:4157:2: ST1003: var fractionCpuReqs should be fractionCPUReqs (stylecheck)
staging/src/k8s.io/kubectl/pkg/describe/describe.go:4158:2: ST1003: var fractionCpuLimits should be fractionCPULimits (stylecheck)
staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret_docker.go:69:2: ST1003: struct field HttpHeaders should be HTTPHeaders (stylecheck)
staging/src/k8s.io/kubectl/pkg/cmd/apply/applyset.go:73:2: ST1003: const V1ApplySetIdFormat should be V1ApplySetIDFormat (stylecheck)
staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go:2244:6: ST1003: var objJson should be objJSON (stylecheck)
staging/src/k8s.io/legacy-cloud-providers/vsphere/nodemanager.go:79:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/legacy-cloud-providers/vsphere/nodemanager.go:80:2: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/legacy-cloud-providers/vsphere/nodemanager.go:84:7: ST1003: type VmSearch should be VMSearch (stylecheck)
staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere.go:1160:5: ST1003: should not use ALL_CAPS in Go names; use CamelCase instead (stylecheck)
staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere.go:1715:20: ST1003: method getZoneByVmUUIDAndNodeName should be getZoneByVMUUIDAndNodeName (stylecheck)
staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_util.go:98:51: ST1003: func parameter nodeVmDetail should be nodeVMDetail (stylecheck)
staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_util.go:133:2: ST1003: var nodeVmDetails should be nodeVMDetails (stylecheck)
staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_util.go:145:9: ST1003: range var nodeVmDetail should be nodeVMDetail (stylecheck)
staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_test.go:841:8: ST1003: range var tagId should be tagID (stylecheck)
staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_test.go:864:8: ST1003: range var tagId should be tagID (stylecheck)
staging/src/k8s.io/metrics/pkg/client/custom_metrics/converter.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/client/custom_metrics/discovery.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/client/custom_metrics/interfaces.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/client/custom_metrics/multi_client.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/client/custom_metrics/versioned_client.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/client/custom_metrics/util_test.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/apis/custom_metrics/conversion.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/apis/custom_metrics/doc.go:21:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/apis/custom_metrics/register.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/apis/custom_metrics/types.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/apis/external_metrics/doc.go:21:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/apis/external_metrics/register.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/apis/external_metrics/types.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/client/external_metrics/client.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/metrics/pkg/client/external_metrics/interfaces.go:17:1: ST1003: should not use underscores in package names (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures.go:62:2: ST1003: should not use underscores in Go names; var baseline_1_0 should be baseline1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures.go:72:2: ST1003: should not use underscores in Go names; var restricted_1_0 should be restricted1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures.go:80:2: ST1003: should not use underscores in Go names; var restricted_1_8 should be restricted1_8 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures.go:89:2: ST1003: should not use underscores in Go names; var restricted_1_19 should be restricted1_19 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures.go:100:2: ST1003: should not use underscores in Go names; var restricted_1_22 should be restricted1_22 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures.go:112:2: ST1003: should not use underscores in Go names; var restricted_1_25_windows should be restricted1_25Windows (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_allowPrivilegeEscalation.go:35:2: ST1003: should not use underscores in Go names; var fixtureData_1_8 should be fixtureData1_8 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_appArmorProfile.go:26:2: ST1003: should not use underscores in Go names; var appArmorFixture_1_0 should be appArmorFixture1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_capabilities_baseline.go:49:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_capabilities_restricted.go:32:2: ST1003: should not use underscores in Go names; var fixtureData_1_22 should be fixtureData1_22 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_hostNamespaces.go:30:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_hostPathVolumes.go:30:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_hostPorts.go:33:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_privileged.go:35:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_procMount.go:26:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_restrictedVolumes.go:26:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_runAsNonRoot.go:39:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_runAsUser.go:39:2: ST1003: should not use underscores in Go names; var fixtureData_1_23 should be fixtureData1_23 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_seLinuxOptions.go:40:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_seccompProfile_baseline.go:32:2: ST1003: should not use underscores in Go names; var fixtureData_baseline_1_0 should be fixtureDataBaseline1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_seccompProfile_baseline.go:68:2: ST1003: should not use underscores in Go names; var fixtureData_baseline_1_19 should be fixtureDataBaseline1_19 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_seccompProfile_restricted.go:32:2: ST1003: should not use underscores in Go names; var fixtureData_restricted_1_19 should be fixtureDataRestricted1_19 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_sysctls.go:33:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_sysctls.go:73:2: ST1003: should not use underscores in Go names; var fixtureData_1_27 should be fixtureData1_27 (stylecheck)
staging/src/k8s.io/pod-security-admission/test/fixtures_windowsHostProcess.go:40:2: ST1003: should not use underscores in Go names; var fixtureData_1_0 should be fixtureData1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_allowPrivilegeEscalation.go:64:6: ST1003: should not use underscores in Go names; func allowPrivilegeEscalation_1_8 should be allowPrivilegeEscalation1_8 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_allowPrivilegeEscalation.go:86:6: ST1003: should not use underscores in Go names; func allowPrivilegeEscalation_1_25 should be allowPrivilegeEscalation1_25 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_appArmorProfile.go:64:6: ST1003: should not use underscores in Go names; func appArmorProfile_1_0 should be appArmorProfile1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_capabilities_baseline.go:62:2: ST1003: should not use underscores in Go names; var capabilities_allowed_1_0 should be capabilitiesAllowed1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_capabilities_baseline.go:79:6: ST1003: should not use underscores in Go names; func capabilitiesBaseline_1_0 should be capabilitiesBaseline1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_capabilities_restricted.go:79:6: ST1003: should not use underscores in Go names; func capabilitiesRestricted_1_22 should be capabilitiesRestricted1_22 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_capabilities_restricted.go:138:6: ST1003: should not use underscores in Go names; func capabilitiesRestricted_1_25 should be capabilitiesRestricted1_25 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_hostNamespaces.go:58:6: ST1003: should not use underscores in Go names; func hostNamespaces_1_0 should be hostNamespaces1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_hostPathVolumes.go:58:6: ST1003: should not use underscores in Go names; func hostPathVolumes_1_0 should be hostPathVolumes1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_hostPorts.go:60:6: ST1003: should not use underscores in Go names; func hostPorts_1_0 should be hostPorts1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_privileged.go:56:6: ST1003: should not use underscores in Go names; func privileged_1_0 should be privileged1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_procMount.go:60:6: ST1003: should not use underscores in Go names; func procMount_1_0 should be procMount1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_restrictedVolumes.go:87:6: ST1003: should not use underscores in Go names; func restrictedVolumes_1_0 should be restrictedVolumes1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_runAsNonRoot.go:61:6: ST1003: should not use underscores in Go names; func runAsNonRoot_1_0 should be runAsNonRoot1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_runAsUser.go:62:6: ST1003: should not use underscores in Go names; func runAsUser_1_23 should be runAsUser1_23 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_seLinuxOptions.go:74:2: ST1003: should not use underscores in Go names; var selinux_allowed_types_1_0 should be selinuxAllowedTypes1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_seLinuxOptions.go:77:6: ST1003: should not use underscores in Go names; func seLinuxOptions_1_0 should be seLinuxOptions1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_seccompProfile_baseline.go:87:6: ST1003: should not use underscores in Go names; func seccompProfileBaseline_1_0 should be seccompProfileBaseline1_0 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_seccompProfile_baseline.go:121:6: ST1003: should not use underscores in Go names; func seccompProfileBaseline_1_19 should be seccompProfileBaseline1_19 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_seccompProfile_restricted.go:69:6: ST1003: should not use underscores in Go names; func seccompProfileRestricted_1_19 should be seccompProfileRestricted1_19 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_seccompProfile_restricted.go:148:6: ST1003: should not use underscores in Go names; func seccompProfileRestricted_1_25 should be seccompProfileRestricted1_25 (stylecheck)
staging/src/k8s.io/pod-security-admission/policy/check_sysctls.go:73:2: ST1003: should not use underscores in Go names; var sysctls_allowed_1_0 should be sysctlsAllowed1_0 (stylecheck)
```

--- a/hack/golangci-strict.yaml
+++ b/hack/golangci-strict.yaml
@@ -15,6 +15,12 @@ issues:
     - path: conversion\.go
       linters:
         - ineffassign
+    # The Kubernetes naming convention for conversion functions uses underscores
+    # and intentionally deviates from normal Go conventions to make those function
+    # names more readable. Same for SetDefaults_*.
+    - linters:
+        - stylecheck
+      text: "ST1003: should not use underscores in Go names; func (Convert_.*_To_.*|SetDefaults_)"
 
 linters:
   disable-all: false # in contrast to golangci.yaml, the default set of linters remains enabled

--- a/hack/golangci.yaml
+++ b/hack/golangci.yaml
@@ -16,6 +16,12 @@ issues:
     - path: (pkg/volume/*|test/*|azure/*|pkg/cmd/wait*|request/bearertoken/*|metrics/*|filters/*) # not in golangci-strict.yaml
       linters:                                                                                    # not in golangci-strict.yaml
         - gocritic                                                                                # not in golangci-strict.yaml
+    # The Kubernetes naming convention for conversion functions uses underscores
+    # and intentionally deviates from normal Go conventions to make those function
+    # names more readable. Same for SetDefaults_*.
+    - linters:
+        - stylecheck
+      text: "ST1003: should not use underscores in Go names; func (Convert_.*_To_.*|SetDefaults_)"
 
 linters:
   disable-all: true # not disabled in golangci-strict.yaml

